{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+exclude: .+/migrations/.+\.py
+default_stages: [commit]
+fail_fast: true
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: master
+    hooks:
+      - id: trailing-whitespace
+        files: (^|/){{cookiecutter.project_slug}}/.+\.(py|html|sh|css|js)$
+
+-   repo: local
+    hooks:
+      - id: black
+        name: black
+        entry: black
+        args: [--line-length=80]
+        language: python
+        types: [python]
+        files: (^|/){{cookiecutter.project_slug}}/.+\.py$
```

Comment:
Looking at the [Black documentation](https://black.readthedocs.io/en/stable/version_control_integration.html), I don't think we need this line:

```suggestion
```